<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="topWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>90</height>
     </rect>
    </property>
    <widget class="QToolButton" name="tbMyPhone">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>70</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbTodayHot">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>70</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbPlayGame">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>70</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbFindGame">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>70</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbApp">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>70</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbAmusement">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>70</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbSmallApp">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>70</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbPcGame">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>70</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbPcApp">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>10</y>
       <width>70</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>25</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDownload">
     <property name="geometry">
      <rect>
       <x>955</x>
       <y>45</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbClose">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>15</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMax">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>15</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMin">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>15</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMenu">
     <property name="geometry">
      <rect>
       <x>905</x>
       <y>15</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMessage">
     <property name="geometry">
      <rect>
       <x>885</x>
       <y>15</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLogin">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>10</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>hi, 请登陆</string>
     </property>
    </widget>
    <widget class="QWidget" name="icon" native="true">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="iconBg" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="SearchWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>45</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <zorder>tbMyPhone</zorder>
    <zorder>tbTodayHot</zorder>
    <zorder>tbPlayGame</zorder>
    <zorder>tbFindGame</zorder>
    <zorder>tbApp</zorder>
    <zorder>tbAmusement</zorder>
    <zorder>tbSmallApp</zorder>
    <zorder>tbPcGame</zorder>
    <zorder>tbPcApp</zorder>
    <zorder>widget</zorder>
    <zorder>pbDownload</zorder>
    <zorder>pbClose</zorder>
    <zorder>pbMax</zorder>
    <zorder>pbMin</zorder>
    <zorder>pbMenu</zorder>
    <zorder>pbMessage</zorder>
    <zorder>pbLogin</zorder>
    <zorder>iconBg</zorder>
    <zorder>icon</zorder>
    <zorder>widget_2</zorder>
   </widget>
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>1000</width>
      <height>610</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>1001</width>
       <height>611</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SearchWidget</class>
   <extends>QWidget</extends>
   <header location="global">searchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
